fuse/data/datasets/dataset_default.py

```diff
@@ -185,7 +185,12 @@ def getitem(
             raise Exception("you must first call create()")
 
         # get sample id
-        if not isinstance(item, (int, np.integer)) or not self._explicit_sample_ids_mode:
+        if not self._explicit_sample_ids_mode:
```

Comment:
Not relevant for this PR, but I suggest we change the flag's name from `_explicit_...` to `_implicit_...` .